### ISSUE TITLE
Question about the step in wandb log

### ISSUE THREAD

**Issue by ziqi-zhang** [FedML-AI/FedML#100]
Hi, thanks for sharing the awesome project. I'm trying the standalone experiments and have a question about the cifar results. I consulted the online result (https://wandb.ai/automl/fedml/runs/2cxam561/) but I don't understand the total steps of the log. According to the plot, there are totally 400 steps, but the configuration is 100 rounds and 20 epochs. I read the code and to my understanding the default evaluation freq is 5 so there should be 20 steps (5 rounds per step) in the log. So Why does the online log have 400 steps? Thanks!

**Comment by chaoyanghe** [comment]
![image](https://user-images.githubusercontent.com/7238845/107428954-2f102700-6ad8-11eb-8098-0057261b33a7.png)
You can edit the x-axis to "round" by clicking the button I highlighted in the picture above

**Comment by ziqi-zhang** [comment]
Thanks for your answer @chaoyanghe . And I have another question about the cifar10 result. I didn't find the result of cifar10 in the standalone FedAvg readme document (https://github.com/FedML-AI/FedML/tree/master/fedml_experiments/standalone/fedavg). I notice in the benchmark that there is a cifar FedAvg log (https://wandb.ai/automl/fedml/runs/2cxam561/) but I tried the hyperparameter setting of this log and only got an acc of 0.35. I ran the code on 1 gpu. Is there anything I did wrong or the result of FedAvg on cifar10 is just as low as 0.35? BTW I notice there is cifar10 result in the standalone FedNova doc.
